### DISCUSSION TITLE
[General] Inter-OP Stack Messaging

### DISCUSSION THREAD

**Discussion by tec05** [ethereum-optimism/developers#299] [General]
### Discussion Topic

I was looking for access to the notion site mentioned in this [discussion](https://github.com/ethereum-optimism/developers/discussions/124) relating to the ability to send messages between OP Stacks.

Ideally looking to see if it is/will be possible to send data (abi encoded) back and forth between different OP Stacks like OP Mainnet and Base as an example.

Any information that could be shared would be tremendously helpful.

Thanks,
Tec

**Comment by hash1go** [comment]
I'm not sure about the access to the Notion site mentioned in that discussion.
However, the development of messaging functionality between OP Stacks is currently in progress. Please have a look here.

https://github.com/ethereum-optimism/specs/tree/main/specs/interop
https://github.com/ethereum-optimism/specs/blob/main/specs/interop/predeploys.md

**Reply by tec05** [reply]
Thanks for the reply and pointer to the specs above! I’ll parse through these to get a lay of the stack.

**Comment by tec05** [comment]
Do you happen to know (in Discord perhaps) where I could go to talk with the team driving this regarding questions and feedback?

**Reply by sbvegan** [reply]
Let me flag this internally and see what we can find out

**Reply by sbvegan** [reply]
You can directly comment in issues and PRs in the [specs repository](https://github.com/ethereum-optimism/specs) with questions and feedback. If you want to start additional discussion, you can follow up in that thread or start a new discussion and we can try and get the folks working on interop to respond to specific questions you have.

**Reply by tec05** [reply]
Great! Thanks @sbvegan!

**Comment by tec05** [comment]
Just following up here. Seemed weird to make a comment in the spec to start additional discussions. Is it possible to put me in touch with the team working on interop? I’d like to talk with them about a reference implementation to showcase interop on the OP stack. So it would be nice to connect with them and see what their plans are for going to testnet and then to mainnet.

Thanks again,
Tec

**Reply by sbvegan** [reply]
Hey @tec05, I'm sorry that I'm just seeing this. I'll try and get some folks on the interop team over here. In the meantime, I'll work on getting a contributing doc in the specs repo so there's a clear way to open up a discussion.
